include/poller.php

```diff
@@ -153,8 +154,24 @@ function poller_execute($queue) {
 	$funcname = str_replace(".php", "", basename($argv[0]))."_run";
 
 	if (function_exists($funcname)) {
+
 		logger("Process ".$mypid." - Prio ".$queue["priority"]." - ID ".$queue["id"].": ".$funcname." ".$queue["parameter"]);
 
+		$stamp = (float)microtime(true);
+
+		if (Config::get("system", "profiler")) {
```

Comment:
a comment for this block wouldn't be bad. something like `// Initialize the profiler`